js/ui/map.js

```diff
@@ -1236,6 +1236,11 @@ function removeNode(node) {
   * Returns a Boolean indicating whether the browser [supports Mapbox GL JS](https://www.mapbox.com/help/mapbox-browser-support/#mapbox-gl-js).
   *
   * @function supported
+  * @param {Object} options
+  * @param {boolean} [options.failIfMajorPerformanceCaveat=false] If `true`,
+  *   the function will return `false` if it is determined
+  *   that the performance of the created WebGL context would be too low.
```

Comment:
Can we use the copy from the deleted documentation instead? It is the product of additional rounds of revision. 

```diff
/**
  * Test if the current browser supports Mapbox GL JS
  * @param {Object} options
  * @param {boolean} [options.failIfMajorPerformanceCaveat=false] Return `false`
  *   if the performance of Mapbox GL JS would be dramatically worse than
  *   expected (i.e. a software renderer would be used)
  * @return {boolean}
  */
```